src/Illuminate/Http/Testing/FileFactory.php

```diff
@@ -28,7 +30,9 @@ public function create($name, $kilobytes = 0)
      */
     public function image($name, $width = 10, $height = 10)
     {
-        return new File($name, $this->generateImage($width, $height));
+        $type = Str::endsWith($name, ['.jpg', '.jpeg']) ? 'jpeg' : 'png';
```

Comment:
I would use `strtolower($name)` because in some cases `image.JPG` might also come.